cloud/awsprovider.go

```diff
@@ -561,3 +619,112 @@ func (*AWS) QuerySQL(query string) ([]byte, error) {
 	return nil, fmt.Errorf("Error getting query results : %s", *qrop.QueryExecution.Status.State)
 
 }
+
+type spotInfo struct {
+	Timestamp   string `csv:"Timestamp"`
+	UsageType   string `csv:"UsageType"`
+	Operation   string `csv:"Operation`
+	InstanceID  string `csv:"InstanceID`
```

Comment:
Having a nice editor is nice
![image](https://user-images.githubusercontent.com/22526/56440802-80b24680-629f-11e9-8321-bcacfa166f64.png)

Reply:
Oops. Thanks for the catch.